app/post_test.go

```diff
@@ -1753,3 +1753,58 @@ func TestCountMentionsFromPost(t *testing.T) {
 		assert.Equal(t, numPosts, count)
 	})
 }
+
+func TestFillInPostProps(t *testing.T) {
```

Comment:
I would add a test case that ensures that the prop is not added if the license does not exist